features/lege-waarden.feature

```diff
@@ -1,10 +1,27 @@
 # language: nl
 
+@post-assert
 Functionaliteit: Leveren van lege waarden
 
   Rule: Een boolean veld met waarde false wordt niet geleverd
```

Comment:
het scenario hieronder gaat toch niet over een boolean maar over een integer (GBA?)

Reply:
titel is denk ik niet duidelijk voor wat je wilt tonen

'indicatie geheim (70.10)' waarde 0 wordt geleverd